cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -1046,6 +1046,13 @@ def report(
         }
     )
 
+    select_cols = [
+        "visit_id",
+        "visit_datetime",
+        *[col for col in valid_df.columns if col.startswith("cis_covid_vaccine_type")],
+    ]
+    other_vaccine_df = valid_df.filter(F.col("cis_covid_vaccine_type") == "Don't know type").select(*select_cols)
```

Comment:
think this filter needs to be where any of the cis_covid_vaccine_type_[n] are 'Don't know type' 

Also the output needs to be what the cis_covid_vaccine_type_other column is in raw. Currently I think this would just show null as we overwrite cis_covid_vaccine_type_other cols to null when recoding cis_covid_vaccine_type cols. Possibly create _orig cols for all cols as part of vaccine cleaning and it would be these cis_covid_vaccine_type_other cols on the reporting output, so we could identify whether strings could be incorporated into regex patterns.